__tests__/app.test.js

```diff
@@ -51,6 +51,7 @@ describe("app", () => {
             topic: "mitch",
             created_at: "2020-10-18T01:00:00.000Z",
             votes: 0,
+            comment_count: "1",
           });
         });
```

Comment:
What about if there are no comments? I think it would be useful to see a test for an article with none (I think article 2 is one of those) and check it comes back as 0 (this is optional and we'd expect a empty arr to come back with 200)

Reply:
✅